CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# 1.1.1
+# 1.2.0
+
+- [upgrade finagle-postgres to 0.4.2](https://github.com/getquill/quill/pull/759)
+- [add collections support for row elements (SQL Arrays, Cassandra Collection)](https://github.com/getquill/quill/pull/758)
+- [add querySchema support to optional nested case class](https://github.com/getquill/quill/pull/767)
+- [upgrade scala to 2.12.2](https://github.com/getquill/quill/pull/770)
```

Comment:
Could you keep it as `Make Quill compatible with Scala 2.12.2` since we support both 2.11/2.12 but not just upgrading to 2.12